index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Totality</title>
+    <meta name="author" content="Anup Kumar Bid" />
+    <meta
+      name="description"
+      content="Totality is a simple CSS frameowrk to kick start your frontend project"
+    />
+    <link rel="stylesheet" href="./src/css/styles.css" />
+  </head>
+  <body>
+    <h1>Style Guide</h1>
+    <hr />
+    <h1>Headings</h1>
+    <hr />
+    <h1>I am a Headline1</h1>
+    <small>H1 48px / Line Height 64px</small>
+
+    <h2>I am a Headline2</h2>
+    <small>H2 40px / Line Height 40px</small>
+
+    <h3>I am a Headline3</h3>
+    <small>H3 32px / Line Height 32px</small>
+
+    <h4>I am a Headline4</h4>
+    <small>H4 24px / Line Height 24px</small>
+
+    <h5>I am a Headline5</h5>
+    <small>H5 16px / Line Height 16px</small>
+
+    <h6>I am a Headline6</h6>
+    <small>H6 8px / Line Height 8px</small>
+
+    <hr />
+    <h1>Text</h1>
+    <hr />
+
+    <p class="has-fs-xxlg has-lh-xxlg">I am a (XXLG) Paragraph</p>
+    <small>P 48px / Line Height 64px</small>
+
+    <p class="has-fs-xlg has-lh-xlg">I am a (XLG) Paragraph</p>
+    <small>P 40px / Line Height 40px</small>
+
+    <p class="has-fs-lg has-lh-lg">I am a (LG) Paragraph</p>
+    <small>P 32px / Line Height 32px</small>
+
+    <p class="has-fs-md has-lh-md">I am a (MD) Paragraph</p>
+    <small>P 24px / Line Height 24px</small>
+
+    <p class="has-fs-sm has-lh-sm">I am a (SM) Paragraph</p>
+    <small>P 16px / Line Height 16px</small>
+
+    <p class="has-fs-xsm has-lh-xsm">I am a (XSM) Paragraph</p>
+    <small>P 8px / Line Height 8px</small>
+
+    <hr />
+    <h1>Avatar</h1>
+    <hr />
+    <img
+      class="avatar is-sm"
```

Comment:
you can give classname avatar-sm instead of is-sm

Reply:
I liked the naming convention for [Bulma](https://bulma.io/) they use this convention so I'm following that. so `.is-sm` is a utility class that can be added to anything whose size needs to be increased/decreased. so if I give Avalar specific classes then the same CSS properties will be there in different classes.